guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p><code>The total time limit in hours for AutoDL to train the networks.</code> to <code>The total time limit in hours for AutoDL to train and evaluate the networks.</code>   </p><p> </p><p>Consider renaming  'perform' to 'Advanced' to be similar to AutoML. </p><p> </p><p><code>To be used when the user wants to tune hyper parameters of two best performing models from basic mode.</code> can be changed to <code>To be used when the user additionally wants to tune hyper parameters of two best performing models from basic mode.</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>